CONTRIBUTING.md

```diff
@@ -16,65 +16,71 @@ We want to make contributing to this project as easy and transparent as possible
 
 
 ### Commit messages
 - Commit messages should be written in a short, descriptive manner and be prefixed with tags for the change type and scope (if possible) according to the [semantic commit](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) scheme.
 For example, a new change to the AIR crate might have the following message: `feat(air): add constraints for the decoder`
 
-- Also would be good to squash commits to logically separated, distinguishable stages to keep git log clean:
-```
-7hgf8978g9... Added A to X \
-                            \  (squash)
-gh354354gh... oops, typo --- * ---------> 9fh1f51gh7... feat(X): add A && B
-                            /
-85493g2458... Added B to X /
+- Also squash commits to logically separated, distinguishable stages to keep git log clean:
+    ```
+    7hgf8978g9... Added A to X \
+                                \  (squash)
+    gh354354gh... oops, typo --- * ---------> 9fh1f51gh7... feat(X): add A && B
+                                /
+    85493g2458... Added B to X /
 
 
-789fdfffdf... Fixed D in Y \
-                            \  (squash)
-787g8fgf78... blah  blah --- * ---------> 4070df6f00... fix(Y): fixed D && C
-                            /
-9080gf6567... Fixed C in Y /
-```
+    789fdfffdf... Fixed D in Y \
+                                \  (squash)
+    787g8fgf78... blah  blah --- * ---------> 4070df6f00... fix(Y): fixed D && C
+                                /
+    9080gf6567... Fixed C in Y /
+    ```
 
 ### Code Style and Documentation
 - For documentation in the codebase, we follow the [rustdoc](https://doc.rust-lang.org/rust-by-example/meta/doc.html) convention with no more than 100 characters per line.
 - For code sections, we use code separators like the following to a width of 100 characters::
-```
-// CODE SECTION HEADER
-// ================================================================================
-```
-[Rustfmt](https://github.com/rust-lang/rustfmt) and [Clippy](https://github.com/rust-lang/rust-clippy) linting included in CI pipeline. Anyways it's prefferable to run linting locally before push:
-```
-cargo fix --allow-staged --allow-dirty --all-targets --all-features; cargo fmt; cargo clippy --workspace --all-targets --all-features -- -D warnings
-```
+    ```
+    // CODE SECTION HEADER
+    // ================================================================================
+    ```
+
+- [Rustfmt](https://github.com/rust-lang/rustfmt) and [Clippy](https://github.com/rust-lang/rust-clippy) linting is included in CI pipeline. Anyways it's prefferable to run linting locally before push:
+    ```
+    cargo fix --allow-staged --allow-dirty --all-targets --all-features; cargo fmt; cargo clippy --workspace --all-targets --all-features -- -D warnings
+    ```
+
+### Versioning
```

Comment:
I suggest to have Versioning at the end, since it might not be relevant for most readers and contributors.